studio/pages/project/[ref]/auth/settings.tsx

```diff
@@ -315,6 +323,8 @@ const Settings = () => {
                   showInlineError
                   errorMessage="Please enter the phone provider."
                 />
+                <NumField name="SMS_OTP_EXP" step="1" min={0}/>
+                <NumField name="SMS_OTP_LENGTH" step="1" min={6} max={10}/>
```

Comment:
any max length?

Reply:
yeah this is the same as the setting in gotrue, min length is 6 and max length is 10